src/bit.c

```diff
@@ -5,7 +5,15 @@
 #include <stdint.h>
 #include <string.h>
 
-#include <arpa/inet.h>
+#if __BYTE_ORDER__ == __ORDER_LITTLE_ENDIAN__
+#	define be_to_host_16(x) (__builtin_bswap16(x))
```

Comment:
Visual Studio doesn't have __builtin_bswap16() nor __builtin_bswap32(). This is why I replaced the memcpy() + conversion with simple functions in https://github.com/matwey/libopenvizsla/pull/6/commits/d31bc8bde4cdc8871af6c842a636553a6ba737a3

Reply:
VS has `_byteswap_ushort` and `_byteswap_ulong` respectively.
I think there is no issue to use them here:
```
#if _MSC_VER
                _byteswap_ushort(x);
#else
                __builtin_bswap16(x);
#endif
```